app/src/main/java/com/yeonae/chamelezone/view/review/ReviewModifyActivity.kt

```diff
@@ -115,18 +112,28 @@ class ReviewModifyActivity :
 
         presenter.getReview(placeNumber, reviewNumber)
 
+        btn_clear.setOnClickListener {
+            uriSet.clear()
+            image_container.removeAllViews()
+            Logger.d("imageNumber $imageNumbers")
+            deleteImageNumber = imageNumbers
+            Logger.d("deleteImageNumber $deleteImageNumber")
+        }
+
         btn_modify.setOnClickListener {
             val content = "${edt_review.text}"
             when {
                 edt_review.text.isEmpty() || edt_review.text.isBlank() -> shortToast(R.string.review_content)
-                uriList.isEmpty() && savedImages.isEmpty() -> shortToast(R.string.review_image)
-                ((imageNumbers.count() - deleteImageNumber.count()) + uriList.count() > 4) -> shortToast(R.string.review_image_max)
+                uriSet.isEmpty() && savedImages.isEmpty() -> shortToast(R.string.review_image)
+                ((imageNumbers.count() - deleteImageNumber.count()) + uriSet.count() > 4) -> shortToast(
+                    R.string.review_image_max
+                )
```

Comment:
```suggestion
                (imageNumbers.count() - deleteImageNumber.count() + uriSet.count() > 4) -> {
                        shortToast(R.string.review_image_max)
                }
```
- + 연산의 순서가 상관 없는 경우이므로 괄호를 제거해도 될 것 같고, 줄바꿈이 혼동을 줄 가능성이 있어보입니다.

Reply:
수정했습니다!
d12f6c5